src/Model bias analysis.ipynb

```diff
@@ -0,0 +1,1554 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model bias analysis\n",
+    "\n",
+    "This notebook uses the bias-fuzzed test sets and the generated bias madlibs dataset to evaluate a model for potential bias."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model loading and inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO(jetpack): rewrite this to use nthain's library\n",
+    "\n",
+    "import cPickle\n",
+    "import os\n",
+    "\n",
+    "from keras.models import load_model\n",
+    "from keras.preprocessing.text import Tokenizer\n",
+    "from keras.preprocessing.sequence import pad_sequences\n",
+    "from keras.utils import to_categorical\n",
+    "\n",
+    "MODEL_VERSION = 'wiki_tox_labels_v0'\n",
+    "MODEL_DIR = '../models/'\n",
+    "\n",
+    "# TODO(nthain): During model building, save relevant hyperparameters and \n",
+    "# load here.\n",
+    "MAX_SEQUENCE_LENGTH = 1000 #Must match the model's\n",
+    "BATCH_SIZE = 128 #Must match the model's\n",
```

Comment:
I was wrong about this. You can remove BATCH_SIZE if you'd like as it shouldn't make a difference.

Reply:
Ah, thanks. I'll clean this up when I rewrite it to use your library.

Reply:
Sounds good!